src/sql/workbench/services/objectExplorer/browser/asyncServerTreeRenderer.ts

```diff
@@ -92,7 +92,7 @@ class ConnectionProfileTemplate extends Disposable {
 		super();
 		container.parentElement!.classList.add('connection-profile');
 		this._root = dom.append(container, dom.$('.connection-tile'));
-		this._icon = dom.append(this._root, dom.$('div.icon server-page'));
```

Comment:
I didn't spend too much time digging out how this could have worked in the past and stopped working recently, if we want to specify 2 classes, we have to use 'div.icon.server-page'.